xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 12.10.2
+JSONRPC_VERSION 12.11.0
```

Comment:
As this will be the first API change for v21, mind just doing a major bump to 13.0.0

Reply:
ofc done

Reply:
For future reference the bump to 13.0.0 here was a mistake. This is not a breaking change, and so minor version change to 12.11.0 was correct. 
The API major version change for v21should happen only at the first breaking change to the API, and to this is best delayed when possible as it prevents and subseqeunt fixes being backported to v20.